server/src/middlewares/pagination.ts

```diff
@@ -5,25 +5,32 @@ export interface PaginationParams {
   offset: number;
 }
 
+interface PaginationRequest {
```

Comment:
Let's rename the directory to `middleware`, or let's rename the other directories to their plural forms.